src/distributions.jl

```diff
@@ -2,62 +2,62 @@
 using Distributions
 
 """
-    MPointvariate
+    FVectorvariate
 
 Structure that subtypes `VariateForm`, indicating that a single sample
-is a point on a manifold.
+is a vector from a fiber of a vector bundle.
 """
-struct MPointvariate <: VariateForm
+struct FVectorvariate <: VariateForm
 end
 
 """
-    FVectorvariate
+    FVectorSupport(space::Manifold, VectorBundleFibers)
 
-Structure that subtypes `VariateForm`, indicating that a single sample
-is a vector from a fiber of a vector bundle.
+Value support for tangent vector-valued distributions (values from tangent
```

Comment:
It's true that at the moment `FVectorSupport` is used only for tangent vectors but the interface is supposed to work for any fiber of any vector bundle.

Reply:
I think I did not change that line. Do you propose to change it? If so, what would be your suggestion?

Reply:
Right, it was like that before. I think something like this would be better:

Value support for vector bundle fiber-valued distributions (values from a fiber of a vector bundle at point `x` from the given manifold).
For example used for tangent vector-valued distributions.